1_Explore_a_Prediction.py

```diff
@@ -3,15 +3,15 @@
 import extra_streamlit_components as stx
```

Comment:
This may have been introduced in a previous PR, but I think we can remove the "Timestamp:" from the options in the timestamp dropdown menu. It doesn't really add info and makes it harder to pick out the actual times